packages/foreman/foreman/foreman.sysconfig

```diff
@@ -1,18 +1,6 @@
-# the location where foreman is installed
-#FOREMAN_HOME=/usr/share/foreman
-
 # the port which foreman web server is running at
 # note that if the foreman user is not root, it has to be a > 1024
-#FOREMAN_PORT=3000
-
-# the user which runs the web interface
-#FOREMAN_USER=foreman
+FOREMAN_PORT=3000
```

Comment:
Technically you could leave these commented out for a smaller diff but it's not that important.